docs/changelog/116531.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 116531
+summary: "[ES|QL] Add a standard deviation function"
```

Comment:
Remove the prefix `[ES|QL]`. The changelog is organized by `area`, which already says it is ES|QL.